dmcontent/govuk_frontend.py

```diff
@@ -119,26 +119,31 @@ def dm_list_input(
     return params
 
 
-def govuk_label(question: Question) -> dict:
-
+def govuk_label(question: Question, is_page_heading: bool = True, **kwargs) -> dict:
```

Comment:
I think we should make `is_page_heading` kwargs only?

```suggestion
def govuk_label(question: Question, *, is_page_heading: bool = True, **kwargs) -> dict:
```

Reply:
Ooh, I don't actually know what this means. I'm guessing the asterisk is placed to handle any non-keyworded args?

So if you were to pass in, say, (question, is_page_heading=False, "A random string"), would that fail or be handled?

Reply:
The syntax is a little obtuse, but the meaning is that you can't accidentally set `is_page_heading` by passing in two arguments. Without the asterisk, `govuk_fieldset(question, question)` would be valid. Granted it's unlikely to happen or be a problem, but better safe than sorry! I find it also makes things clearer when people write the name of the keyword argument (rather than `govuk_fieldset(question, True)`. What is true doing there?) and this change enforces that.

This feature was added in [Python Enhancement Proposal 3102](https://www.python.org/dev/peps/pep-3102/), but this article explains it a little better https://python-3-for-scientists.readthedocs.io/en/latest/python3_advanced.html#keyword-only-arguments.

Reply:
Incidentally the example you gave would fail either way.

Reply:
So in answer to your question, the `*` syntax prevents any further positional arguments from being read after that point in the argument list.